apps/cookbook/src/app/examples/header-example/examples/action-group.ts

```diff
@@ -11,12 +11,12 @@ import { BasePageExampleComponent } from '../../page-example/base-page-example.c
 const config = {
   template: `<kirby-page defaultBackHref="/">
   <kirby-header [title]="'Action Group'" subtitle1="Subtitle one" subtitle2="Subtitle two">
-    <kirby-action-group *kirbyHeaderActions>
+    <kirby-action-group *kirbyHeaderActions [visibleActions]="3">
       <button kirby-button attentionLevel="3" (click)="actionClicked('Action 1')">
         <kirby-icon name="edit"></kirby-icon>
         <span class="text">Action 1</span>
       </button>
-      <button kirby-button attentionLevel="3" (click)="actionClicked('Action 2')">
+      <button kirby-button attentionLevel="3" (click)="actionClicked('Action 2')" [disabled]="disabled">
```

Comment:
```suggestion
      <button kirby-button attentionLevel="3" (click)="actionClicked('Action 2')" [disabled]="true">
```

Consider not changing visibleActions here, as this is the default example for action group, and I think we should keep it as simple as possible.

Consider adding a separate example, or alternatively validate the behavior with visual and unit testing, and then just write out that buttons can be disabled 😊

Reply:
I think that the disabled state is such a normal thing to expect - that we don't need an example to prove it. I have decided to test it by unit testing